### PR TITLE
Cherry pick win arm64

### PR DESCRIPTION
**Pull request by nsait-linaro**

### CODE REVIEW

**Comment by meteorcloudy** [comment]
I guess you need two more commits to make the tests pass:

- https://github.com/bazelbuild/bazel/commit/095f1e24ae3fa0860bdda5af7befe74c1123a078 this one was submitted between adding Windows ARM64 support and fixing the Java toolchain
- https://github.com/bazelbuild/bazel/commit/f936e1b3ba99a2dfc757339d2d5225338449c65a (for fixing the verify_workspace test)

File: src/test/shell/bazel/verify_workspace.sh [hunk 1]
```diff
@@ -46,14 +46,18 @@ function test_verify_urls() {
...
+  checked_urls=()
   for file in "${WORKSPACE_FILES[@]}"; do
     for url in $(grep -E '"https://|http://' "${file}" | \
       sed -e '/jekyll-tree/d' -e '/^#/d' -r -e  's#^.*"(https?://[^"]+)".*$#\1#g' | \
       sort -u); do
-      #echo "Checking ${url}"
-      if ! curl --head -silent --fail --output /dev/null --retry 3 "${url}"; then
-        #fail "URL ${url} is invalid."
-        invalid_urls+=("${url}")
+      # add only unique url to the array
```
Comment (aiuto) [inline src/test/shell/bazel/verify_workspace.sh:54]: The sort -u should be keeping things unique. You should investigate why it is not and fix that instead.

File: src/main/java/com/google/devtools/build/lib/analysis/config/AutoCpuConverter.java [hunk 2]
```diff
@@ -50,8 +50,10 @@ public String convert(String input) throws OptionsParsingException {
           switch (CPU.getCurrent()) {
             case X86_64:
               return "x64_windows";
+            case AARCH64:
+              return "arm64_windows";
             default:
-              // We only support x64 Windows for now.
+              // We only support x64 and arm64 Windows for now.
```
[the remark this answers is not in the document]
Comment (nsait-linaro) [reply src/main/java/com/google/devtools/build/lib/analysis/config/AutoCpuConverter.java:56]: Ok. https://github.com/bazelbuild/bazel/pull/14804 to fix in master

**Comment by nsait-linaro** [comment]
@meteorcloudy Is this good to go ?

**Review by meteorcloudy — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**